src/wp-includes/pluggable.php

```diff
@@ -2694,6 +2694,7 @@ function get_avatar( $id_or_email, $size = 96, $default = '', $alt = '', $args =
 			'force_display' => false,
 			'loading'       => null,
 			'extra_attr'    => '',
+			'decoding'      => 'async',
```

Comment:
Should this be added below 'loading' instead of after 'extra_attr'?

Confidence check me on this: As these are run through `wp_parse_args()`, this shouldn't introduce a BC break, right?

Reply:
> Confidence check me on this: As these are run through `wp_parse_args()`, this shouldn't introduce a BC break, right?

That's correct.

If a developer is defining their own `decoding` attribute in the function call, that will trump the one that is defined here.